tests/level2_test.go

```diff
@@ -0,0 +1,84 @@
+package caf_tests
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/azure"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSharedServicesResourceGroupsExists(t *testing.T) {
+	t.Parallel()
+
+	test := prepareTestTable()
+
+	client, _ := azure.GetResourceGroupClientE(test.SubscriptionID)
+
+	var top int32 = 100
+
+	result, _ := client.List(context.Background(), "tagName eq 'level' and tagValue eq 'level2'", &top)
```

Comment:
This should also check that landingzone : shared_services. We want to ensure that this test only factors in the shared services lz and not launchpad. So we should be adding the condition and update expected value.

Reply:
updated test, thanks for the feedback